testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py

```diff
@@ -57,13 +63,22 @@ def removeFile(fileName):
 print "running", ' '.join(args)
 subprocess.check_call(args, env=os.environ.copy())
 
-print 'Step 6. Injecting bathymetry'
-args = ['./inject_bathymetry.py',
-        'base_mesh.nc']
-print "running", ' '.join(args)
-subprocess.check_call(args, env=os.environ.copy())
+if cl_args.inject_bathymetry:
+  print 'Step 6. Injecting bathymetry'
+  args = ['./inject_bathymetry.py',
+          'base_mesh.nc']
+  print "running", ' '.join(args)
+  subprocess.check_call(args, env=os.environ.copy())
+
+if cl_args.inject_cullCell > 0.0:
+  print 'Step 7. Injecting flag to preserve floodplain'
+  args = ['./inject_preserve_floodplain.py',
+          'base_mesh.nc',          
+          str(cl_args.inject_cullCell)]
```

Comment:
This would need changed too if `inject_cullCell` is removed, and I would suggest it should be given that it has an associated argument that is a float and not a bool.

Reply:
I changed this as well in the latest commit.